explainers/prioritized-task-scheduling.md

```diff
@@ -0,0 +1,856 @@
+console.log(signal instanceof AbortSignal);  // true
+console.log(signal.aborted);                 // false
+```
+
+**Example: Signaling 'prioritychange' and 'abort'.**
+```js
+const controller = new TaskController({priority: 'background'});
+const signal = controller.signal;
+// TaskSignal fires 'prioritychange' events when the priority changes.
+signal.addEventListener('prioritychange', handler);
+controller.setPriority('user-visible');
+console.log(signal.priority);                // 'user-visible'
+
+// TaskController can abort the associated TaskSignal.
+controller.abort();
+console.log(signal.aborted);                 // true
+```
+
+
+#### `TaskSignal.any()` Specialization
+
+(See also the [`TaskSignal.any()` explainer](https://github.com/shaseley/abort-signal-any#tasksignal-apis)
+and [specification](https://wicg.github.io/scheduling-apis/#dom-tasksignal-any).)
+
+[`AbortSignal.any()`](https://dom.spec.whatwg.org/#dom-abortsignal-any) creates an `AbortSignal`
+that is aborted when any of the signals passed to it are aborted. We call this a _dependent signal_
+since it is dependent on other signals for its abort state.
+
+[`TaskSignal.any()`](https://wicg.github.io/scheduling-apis/#dom-tasksignal-any) is a specialization
+of this (inherited) method. It returns a `TaskSignal` which is similarly aborted when any of the
+signals passed to it are aborted, but additionally it has priority, which by default is
+"user-visible" but can be customized &mdash; either to a fixed priority or a dynamic priority based
+on another `TaskSignal`.
+
+Summarizing, compared to `AbortSignal.any()`, `TaskSignal.any()`:
+ - Returns a `TaskSignal` instead of an `AbortSignal`
+ - Has the same behavior for abort, i.e. it is dependent on the input signals for its abort state
+ - Also initializes the priority component of the signal, to either a fixed priority or a dynamic
+   priority based on an input `TaskSignal` (meaning it changes as the input signal changes)
+
+**Example: `TaskSignal.any()` with default priority.**
+```js
+// The following behaves identical to AbortSignal.any([signal1, signal2]), but
+// the signal returned is a TaskSignal with default priority.
+const signal = TaskSignal.any([signal1, signal2]);
+console.log(signal instanceof TaskSignal);  // true
+console.log(signal.priority);               // 'user-visible' (default)
+```
+
+**Example: `TaskSignal.any()` with a fixed priority.**
+```js
+// The resulting signal can also be created with a fixed priority:
+const signal = TaskSignal.any([signal1, signal2], {priority: 'background'});
+console.log(signal.priority);               // 'background'
+```
+
+**Example: `TaskSignal.any()` with a dynamic priority.**
+```js
+// Here, `signal` is dependent on `controller.signal` for priority.
+const controller = new TaskController();
+const sourceSignal = controller.signal;
+
+const signal = TaskSignal.any([signal1, sourceSignal], {priority: sourceSignal});
```

Comment:
Is it important here that `sourceSignal` also appears in the list of signals passed to `.any()`? Would

```
const signal = TaskSignal.any([signal1, signal2], {priority: sourceSignal});
```

work as well? (I don't think you should change it to this, but if that *doesn't* work, then we could call out that `sourceSignal` must also be present in the list)